images/base/files/etc/systemd/system/containerd-fuse-overlayfs.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=containerd fuse-overlayfs snapshotter
+PartOf=containerd.service
+
+[Service]
+ExecStart=/usr/local/bin/containerd-fuse-overlayfs-grpc /run/containerd-fuse-overlayfs.sock /var/lib/containerd-fuse-overlayfs
+Type=notify
```

Comment:
does this means this is always running?
is socket based activation an option?

Reply:
The `containerd-fuse-overlayfs.service` is disabled by default. The service is enabled when `entrypoint` detects that kernel-mode overlayfs is unavailable.

The `Type=notify` line just means that the `containerd-fuse-overlayfs.service` can talk `sd_notify` API.
Socket-based activation is unlikely to be an option, as the service needs to be always running when there is an active container that uses `fuse-overlayfs`.

Reply:
:+1: